src/Elmish.WPF/InternalUtils.fs

```diff
@@ -68,6 +68,29 @@ module ValueOption =
     | Ok x -> ValueSome x
     | Error _ -> ValueNone
 
+  [<RequireQualifiedAccess>]
+  type ToNullError =
```

Comment:
So now you are going to handle these cases in your production code?

Reply:
Yes. This is a failure case in the production case, same as if `Initialize` returns `None`

Reply:
See [PR 470 `DynamicViewModel.fs:122-123`](https://github.com/elmish/Elmish.WPF/pull/470/files#diff-f89d990c1c18eeba924956d01fdbc27ccafa1169853c7d35a040a28f76225ec9R122-R123)

Reply:
Thanks.  When we get there, I would like to see test that execute those lines.